src/CMakeLists.txt

```diff
@@ -29,8 +29,25 @@ string(JOIN "\n" CELERITAS_RNG_MACROS
   "#define CELERITAS_RNG CELERITAS_RNG_${CELERITAS_RNG}"
 )
 configure_file("celeritas_config.h.in" "celeritas_config.h" @ONLY)
+
+#----------------------------------------------------------------------------#
+
+# TODO: add build flags
+set(CELERITAS_CMAKE_STRINGS)
+set(CELERITAS_BUILD_TYPE ${CMAKE_BUILD_TYPE})
+foreach(_var CELERITAS_RNG CELERITAS_BUILD_TYPE)
+  string(TOLOWER "${_var}" _lower)
+  list(APPEND CELERITAS_CMAKE_STRINGS
+    "static const char ${_lower}[] = \"${${_var}}\";"
+  )
+endforeach()
```

Comment:
Maybe this?
```suggestion
    "static const char ${_lower}[] = \"${${_var}}\"\;"
  )
endforeach()

string(JOIN "\n" CELERITAS_CMAKE_STRINGS
  ${CELERITAS_CMAKE_STRINGS}
)
```

Reply:
Uh oh, it's "working" on my machine at present because semicolon is the CMake list separator -- which makes me realize I'm going to have to treat this like a string instead of using a list-join, since that will remove the end-of-line semicolons as well 😂  I'll get to this later.